.github/workflows/build.yml

```diff
@@ -0,0 +1,84 @@
+name: Build WordPress
+
+on:
+  push:
+    branches:
+      - trunk
+      - '3.[7-9]'
```

Comment:
Thinking about this more as well. Do we want artifacts to be present for every commit and every branch/tag going forward? If so, then this is fine. But if we only need them for pull requests right now, we should remove the `push` event here, and probably the branch filters below for `pull_request`.